config/core/build-configs.yaml

```diff
@@ -262,6 +262,21 @@ fragments:
       - 'CONFIG_USER_NS=y'
       - 'CONFIG_BLK_DEV_NVME=y'
 
+  x86-chromebook-full:
+    path: "kernel/configs/x86-chromebook-full.config"
+    configs:
+      - 'CONFIG_CROS_EC=m'
+      - 'CONFIG_CROS_EC_LPC=m'
+      - 'CONFIG_IIO=m'
+      - 'CONFIG_IIO_CROS_EC_SENSORS_CORE=m'
+      - 'CONFIG_IIO_CROS_EC_SENSORS=m'
```

Comment:
Is there a good reason to build them as modules rather than having them built-in?

Reply:
Usually having as module give you more chances to catch issues, I was solving in the past lots of issues and dependencies problems that were only noticed when the driver is built as module.

Said that, I'd follow the same rule that is followed in other defconfigs (like the arm64 defconfig), try to use always modules unless the option is really needed to boot (i.e some mmc controllers, etc.). In the case of cros-ec, some Chomebooks has problems booting unless is built-in, i.e when the cros-ec controls a regulator that is used for the emmc. I'd propose:

'CONFIG_CROS_EC=y'
'CONFIG_CROS_EC_LPC=y'
'CONFIG_IIO=m'
'CONFIG_IIO_CROS_EC_SENSORS_CORE=m'
'CONFIG_IIO_CROS_EC_SENSORS=m'

I'm missing, though, CONFIG_MFD_CROS_EC_DEV

Note that these options should be needed to increase coverage of the cros-ec-tests done (non chromeos userspace dependent). And that should be the argument to include these configs in this fragment.

@aratiu: Would be nice to see a couple of jobs running the cros-ec-tests with and without this fragment, to see  that there is a difference.

Reply:
I've pushed the new version and also added links to some runs on LAVA boards for cros-ec-tests.